### PR TITLE
Update fees.md

### PR DESCRIPTION
**Pull request by KshitijChaudhary666**
Updating with collapsible information on transaction fee on evm

### CODE REVIEW

File: docs/evm/fees.md [hunk 1]
```diff
@@ -9,13 +9,85 @@ sidebar_position: 4
...
 
 :::
-
-# Fees
+<h2>Fees</h2>
 
 EVM transactions are ultra low-cost and use the native FLOW token as gas. [Externally Owned Accounts (EOAs)](https://developers.flow.com/evm/build/accounts) function the same on Flow as other EVM networks like Ethereum.
 
-**Gasless Transactions**
+<details>
```
Comment (vishalchangrani) [inline docs/evm/fees.md:16]: I think we are overdoing it with the expandable sections :)
If you look at the preview - https://docs-psucmflu5-onflow.vercel.app/evm/fees, the whole page is nothing but expandable sections (very different from the other pages under EVM).

Lets have this section as a regular section.

File: docs/evm/fees.md [hunk 2]
```diff
@@ -9,13 +9,85 @@ sidebar_position: 4
...
+
+```
+Transaction fee on EVM = surge x [inclusion fee + (execution effort * unit cost)] 
+```
+- `Surge' factor` dynamically accounts for network pressure and market conditions. This is currently constant at 1.0 but subject to change with community approval.
+- `Inclusion fee` accounts for the resources required to process a transaction due to its core properties (byte size, signatures). This is currently constant at 1E-6 FLOW, but subject to change with community approval.
+- `Execution fee` The fee that accounts for the operational cost of running the transaction script, processing the results, sending results for verification, generating verification receipts, etc. and is calculated as a product of `execution effort units` and the `cost per unit`.
+  - `Execution Effort (computation)` is based on transaction type and operations that are called during the execution of a transaction. The weights determine how “costly” (time consuming) each operation is.
```
Comment (vishalchangrani) [inline docs/evm/fees.md:32]: Lets move the "Calculation of execution effort" expandable section here.

**Review by vishalchangrani — APPROVED** [review]
left some comments about the layout - approving this PR.
You can make the change as part of this PR or a separate PR.

**Comment by KshitijChaudhary666** [comment]
> left some comments about the layout - approving this PR. You can make the change as part of this PR or a separate PR.

Thanks. Made the changes to the same PR

**Pull request merged**